package.json

```diff
@@ -113,5 +116,8 @@
   },
   "bin": {
     "contentful-migration": "./bin/contentful-migration"
+  },
+  "engines": {
+    "node": ">=8.0.0"
```

Comment:
After replacing `Object.value` with `_,values` we could target also node v6. Is there anything else preventing us from doing so?

Reply:
Currently we have set the compile target to `es2017`, mainly for cleaner stack traces.
When not supporting Node 6, we get rid of all the `Generator` lines in the trace.

Reply:
We do not need to turn async/await into generators then